README.md

```diff
@@ -60,6 +60,7 @@ pre-commit install  # for clean and tidy code
 ```
 
 ## 🔥 News
+* 2024.05.15: Add support for score calibration, see [#320](https://github.com/wenet-e2e/wespeaker/pull/320)
```

Comment:
add hyper-links for score calibration:
https://arxiv.org/pdf/2010.11255 or our: https://arxiv.org/pdf/2211.00815